CMakePresets.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Do we need to pull in any of the `bde-tools` toolchain files here, either [by chainloading them](https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration#using-multiple-toolchain-files) or copying some of the configuration into `base`?

Reply:
No, the toolchains provided by bde-tools haven't been necessary so far. We may consider using chainloading, but what vcpkg's release and override triplet features provide are a fine enough substitute.

Reply:
Okay, makes sense.